interactive/views.py

```diff
@@ -195,66 +193,48 @@ def get_object(self, queryset=None):
 
     def get_context_data(self, **kwargs):
         report = process_html(self.object.report_content)
-        update_url = reverse(
-            "interactive:report-update",
-            kwargs=dict(
-                slug=self.object.slug,
-                project_slug=self.object.project.slug,
-                org_slug=self.object.project.org.slug,
-            ),
-        )
+        edit_url = self.object.get_edit_url()
 
         return super().get_context_data(**kwargs) | {
             "report": report,
-            "update_url": update_url,
+            "edit_url": edit_url,
```

Comment:
We can get this directly off the `report` instance in the template now :)  (that was part of the reason I suggested it!)

Reply:
oh! nice